src/cljs/rems/application.cljs

```diff
@@ -406,6 +406,14 @@
       :always (into [:div
                      [:div.mb-3 {:class (str "state-" (name state))}
                       (phases (get-application-phases state))]
+                     [info-field
+                      (text :t.applications/application)
+                      [:span#application-id (:application/external-id application)]
+                      {:inline? true}]
+                     [info-field
+                      (text :t.actions/description)
```

Comment:
I think the general look and feel of Application page is greatly improved by these additions.